entrypoint.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh -l
+set -euo pipefail
+
+echo "setting prerequisites."
+export BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | tr -d '[:punct:]')
+
+
+if [ $BRANCH_NAME != 'master' ];
+then 
+    export VERSION_SUFFIX=$BRANCH_NAME.$GITHUB_RUN_NUMBER
+else
+    export VERSION_SUFFIX=$BRANCH_NAME.$GITHUB_RUN_NUMBER.1
+fi
+
+echo "Print Branch name : $BRANCH_NAME"
+echo "Print VERSION_SUFFIX1: $VERSION_SUFFIX"
+
+echo "Current directory : $GITHUB_WORKSPACE"
+
+echo "Start: code build."
+if [ -z "${CUSTOM_DOCKER_FILE_PATH}" ];
+then
+   echo "Running lazy build steps."
+   cd $GITHUB_WORKSPACE && dotnet build --configuration Release
+   echo "Start run tests."
+   cd $GITHUB_WORKSPACE && dotnet test
```

Comment:
sonar scan also needs to do a test. can you merge both?

Reply:
Updated